### DISCUSSION TITLE
Is there going to be a Safari version coming soon?

### DISCUSSION THREAD

**Discussion by thejakecampbell** [uBlockOrigin/uBlock-issues#3197] [General] [answered]
With most other adblockers being paid or having privacy concerns, is there going to be a Safari version developed or ported to the Mac anytime soon? I think many others would love to see this come to life as well.

**Comment by garry-ut99** [comment]
There was support for Safari in the past, several years ago, but not anymore, and it's unknown whether will be again or not.

Duplicate of (main discussion threads):
- https://github.com/el1t/uBlock-Safari/issues/158
- https://github.com/uBlockOrigin/uBOL-home/issues/52
- https://github.com/gorhill/uBlock/issues/1596

<details>
<summary>Also duplicate of many single questions:</summary>

- https://github.com/uBlockOrigin/uBlock-issues/issues/2841
- https://github.com/uBlockOrigin/uBlock-issues/issues/1127
- https://github.com/uBlockOrigin/uBlock-issues/issues/1123
- https://github.com/uBlockOrigin/uBlock-issues/issues/260
- https://github.com/el1t/uBlock-Safari/issues
- https://www.reddit.com/r/Safari/comments/17u64hc/converting_ublock_to_run_on_safari/
- https://www.reddit.com/r/Safari/comments/139s2d4/thinking_to_bring_back_ublock_origin_for_safari/
- https://www.reddit.com/r/Safari/comments/11lc416/ublock_origin_for_safari/
- https://www.reddit.com/r/Safari/comments/8z3wfz/is_ublock_origin_for_safari_dead/

(from https://www.reddit.com/r/uBlockOrigin):
- https://www.reddit.com/r/uBlockOrigin/comments/1agl8oq/will_there_ever_be_ubo_for_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/18n98to/ubo_ios_for_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/18fhlz6/current_status_of_ublockorigin_in_safari_17/
- https://www.reddit.com/r/uBlockOrigin/comments/14kgnjv/ublock_origin_for_safari_17/
- https://www.reddit.com/r/uBlockOrigin/comments/12ewrn9/are_there_any_updates_on_safari_support/
- https://www.reddit.com/r/uBlockOrigin/comments/1245u0w/lots_of_webkit_features_added_to_safari_164_will/
- https://www.reddit.com/r/uBlockOrigin/comments/ybri7q/is_there_a_version_of_ublockorigin_for_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/xmnqs2/ubo_lite_and_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/x6h7fb/will_ublockorigin_support_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/v6mgc7/is_there_is_a_chance_of_ubo_coming_to_ios_16_as/
- https://www.reddit.com/r/uBlockOrigin/comments/sf0rjp/does_anybody_want_to_publish_ublock_origin_on/
- https://www.reddit.com/r/uBlockOrigin/comments/re5y0i/safari_preview_adds_new_supported_features_are/
- https://www.reddit.com/r/uBlockOrigin/comments/qhwya5/ublock_origin_on_safari_15_macos/
- https://www.reddit.com/r/uBlockOrigin/comments/q2xyax/now_that_safari_supports_extensions_on_ios_will/
- https://www.reddit.com/r/uBlockOrigin/comments/pvi9me/ubo_extensions_for_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/p5lw3f/ublock_origin_for_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/oq2ney/ublock_origin_for_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/nww2uq/safari_web_extension_improvements/
- https://www.reddit.com/r/uBlockOrigin/comments/nqi0pq/will_support_safari_browser_again/
- https://www.reddit.com/r/uBlockOrigin/comments/jxvod5/safari_14/
- https://www.reddit.com/r/uBlockOrigin/comments/heub50/will_ublockorigin_come_to_the_new_safari_in_macos/
- https://www.reddit.com/r/uBlockOrigin/comments/fomths/any_safari_users_out_there_still_wanting_to_use/
- https://www.reddit.com/r/uBlockOrigin/comments/e59w9a/comeback_to_safari_macos_catalina/
- https://www.reddit.com/r/uBlockOrigin/comments/d18cpi/explanation_of_the_state_of_ublock_origin_and/
- https://www.reddit.com/r/uBlockOrigin/comments/cvcb9z/how_do_i_install_this_for_safari/
- https://www.reddit.com/r/uBlockOrigin/comments/cdk6q1/ublock_origin_safari_extension_doesnt_work/
- https://www.reddit.com/r/uBlockOrigin/comments/cavjzg/safari_13_macos_catalina_1015_support/
- https://www.reddit.com/r/uBlockOrigin/comments/bzssz2/is_ublocksafari_dead/
- https://www.reddit.com/r/uBlockOrigin/comments/bvnoyx/safari_extension_still_on_1160_why_doesnt_it/
- https://www.reddit.com/r/uBlockOrigin/comments/bh08t5/is_this_project_dead_for_safari_macos_if_it_is/
- https://www.reddit.com/r/uBlockOrigin/comments/9i08xd/is_there_an_effort_to_publish_an_extension_on_the/


</details>

Use github search functionality to avoid spamming with same questions.

**Reply by thejakecampbell** [reply]
Sorry — I did a search and nothing relevant popped up.

**Reply by garry-ut99** [reply]
Cheap excuse, try to search better, I did search 5 minutes and all the threads mentioned above easily popped up.

**Comment by gothic-bum** [comment, marked as answer]
https://github.com/uBlockOrigin/uBOL-home

https://apps.apple.com/us/app/ublock-origin-lite/id6745342698
